src/7-to-8/summary.rst

```diff
@@ -330,7 +335,10 @@ Cylc Install
 
 .. seealso::
 
-   :ref:`Moving to Cylc Install<majorchangesinstall>`.
+   * :ref:`Moving to Cylc Install<majorchangesinstall>`.
+   * :ref:`tutorials.furthertopics` have been re-written to use cylc install.
+   * :ref:`tutorial.furthertopics.queues` provides an example of using
+     named runs.
```

Comment:
Need to keep this summary quick and simple, if there are too many links people won't follow them.

If `majorchangesinstall` does not contain enough info it needs to be added there.

```suggestion
```